wire-kotlin-generator/src/main/java/com/squareup/wire/kotlin/KotlinGenerator.kt

```diff
@@ -377,12 +377,10 @@ class KotlinGenerator private constructor(
         parameterSpec.defaultValue(defaultValue)
       }
 
-      if (javaInterOp) {
-        parameterSpec.addAnnotation(AnnotationSpec.builder(WireField::class)
-            .addMember("tag = %L", field.tag())
-            .addMember("adapter = %S", getAdapterName(field))
-            .build())
-      }
+      parameterSpec.addAnnotation(AnnotationSpec.builder(WireField::class)
+          .addMember("tag = %L", field.tag())
+          .addMember("adapter = %S", getAdapterName(field))
+          .build())
```

Comment:
Should the annotation be limited to the field element via [use-site targets](https://kotlinlang.org/docs/reference/annotations.html#annotation-use-site-targets)?

Reply:
Yeah, guess that makes sense